lib/sshkit/configuration.rb

```diff
@@ -29,6 +29,10 @@ def default_runner
       @default_runner ||= :parallel
     end
 
+    def default_runner_config
+      @default_runner_config ||= {}
```

Comment:
@robd Thanks for your suggestions.  I added 8014f6d58ef3f136503df8fadfa9a706f3f5a23f, that uses the value of the `default_runner` as the value of the `:in` key.  This also allows a user to set the `default_runner` via the `default_runner_config`, although via the unintuitive `:in` key.